docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,182 @@ Requirements
 The CLI requires a Java virtual machine available on the path.
 It can be used with Java version 8 and higher.
 
+The CLI uses the HTTP protocol and the :doc:`Trino client REST API
```

Comment:
```suggestion
The CLI uses the :doc:`Trino client REST API
```

HTTP is implementation detail of the Trino REST API, may not always be so. If we want to keep it maybe word as `The CLI uses the Trino client REST API over HTTP/S`?

Reply:
lets reword as suggested with HTTP/HTTPS 

the reason we want to have this is that it lets users understand that they just use the http://... URL for their coordinator .. and not some other URL or endpoint and protocol...